### PR TITLE
Add `5 Digit World Cup 2024 Concludes` news post

### PR DESCRIPTION
**Pull request by Walavouchey**
[web preview](https://osu2.roanh.dev/home/news/2024-05-04-5-digit-world-cup-2024-concludes)

- [x] credits image
- [x] two recap sections
- [x] upload original song previews
- [x] redate and retime

### CODE REVIEW

File: news/2024/2024-04-28-5-digit-world-cup-2024-concludes.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+> 
+> Alongside BlankTap's frankly perfect commentary and a map that came down to a 28k score difference on 2.5 million team scores it made for an insanely emotional moment that really reminds me how much I love this game and the community that comes with it <3
+
+Where might you go from here?
+
+> I still plan on hosting and pooling consistently! I plan on sticking around for next year's 5WC as a staff member and will also be continuing the badged [All-American Auction House](https://osu.ppy.sh/community/forums/topics/1692273?n=1) series that has already had 2 successful iterations with the 3rd coming in June. I also plan on starting another badged international series sometime this fall with a fun gimmick that hasn't been seen yet. I stream on twitch a few times a week and I'm open to any questions regarding hosting/pooling either there or in discord dms, and if anyone needs staffing help for a tournament then please feel free to reach out and I'll try to help!
```
Comment (0x84f) [inline news/2024/2024-04-28-5-digit-world-cup-2024-concludes.md:227]:
```suggestion
> I still plan on hosting and pooling consistently! I plan on sticking around for next year's 5WC as a staff member and will also be continuing the badged [All-American Auction House](https://osu.ppy.sh/community/forums/topics/1692273?n=1) series that has already had 2 successful iterations with the 3rd coming in June. I also plan on starting another badged international series sometime this fall with a fun gimmick that hasn't been seen yet. I stream on twitch a few times a week, and I'm open to any questions regarding hosting/pooling either there or in discord dms, and if anyone needs staffing help for a tournament then please feel free to reach out and I'll try to help!
```

**Review by 0x84f — COMMENTED** [review]
a small amount of wording changes, but mostly grammar
also the post will need to be re-timed so might as well put that into the task list

File: news/2024/2024-05-03-5-digit-world-cup-2024-concludes.md [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+
+### Round of 16: ::{ flag=PL }:: Poland vs. ::{ flag=AU }:: Australia
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/qY9OcD7yj3A" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Moving into the Round of 16, we see what looks like an impossible matchup for Australia (#17). To move into Quarterfinals in the winners bracket, they must win against Poland (#1). The absurdity of this match cannot be understated.
+
+In the beginning, all looked normal for Poland, the 98.6% favourite according to the week's [pick'em challenge](/wiki/shared/news/2024-05-03-5-digit-world-cup-2024-concludes/pickem.png), as they took an easy two points from Australia. This is the behavior you expect from a #1 seed in the Round of 16, especially against a team who made it here via an upset against Sweden (#16).
```
Comment (0x84f) [inline news/2024/2024-05-03-5-digit-world-cup-2024-concludes.md:47]:
```suggestion
In the beginning, all looked normal for Poland, the 98.6% favourite according to the week's [pick'em challenge](/wiki/shared/news/2024-05-03-5-digit-world-cup-2024-concludes/pickem.png), as they took an easy two points from Australia. This is the behaviour you expect from a #1 seed in the Round of 16, especially against a team who made it here via an upset against Sweden (#16).
```

File: news/2024/2024-05-03-5-digit-world-cup-2024-concludes.md [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+
+As the match charges on, Australia managed to earn point after point. Expecting an easy 3-0 lead, Poland picked HR3 and lost by a huge margin (by nearly 500k points). The score was now 2-1 and the ball was in Australia's court. Looking to tie things up, the Aussies went for FM2 and managed to win by 300k - a second victory in a row over the goliath that is the Polish team. Everyone's pick'ems were already shattered by this point, but the Australian team still had more up their sleeves and the score is tied 2-2.
```
Comment (0x84f) [inline news/2024/2024-05-03-5-digit-world-cup-2024-concludes.md:49]:
```suggestion
As the match charges on, Australia managed to earn point after point. Expecting an easy 3-0 lead, Poland picked HR3 and lost by a huge margin (by nearly 500k points). The score was now 2-1 and the ball was in Australia's court. Looking to tie things up, the Aussies went for FM2 and managed to win by 300k — a second victory in a row over the goliath that is the Polish team. Everyone's pick'ems were already shattered by this point, but the Australian team still had more up their sleeves and the score is tied 2-2.
```

File: news/2024/2024-05-03-5-digit-world-cup-2024-concludes.md [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+
+Lagging behind and desperately in need of momentum, Poland picked DT3. After an uncertain ending with ::{ flag=AU }:: [Trainjumper](https://osu.ppy.sh/users/2272587)'s client freezing on the livestream, the results were revealed in the [match logs](https://osu.ppy.sh/community/matches/113190844). Australia won again and now had the advantage at 2-3. They shot for NM2 and won yet another point, somehow flipping the script and making Australia the one with match point against the would-be winners of the tournament.
+
+In the end, the run of a lifetime dissolved as Poland locked in and surges for victory, ultimately defeating the Australians in a 5-4 tiebreaker victory. The fact this match went to tiebreaker at all is incredible, and we're certain the Australian team is proud of their efforts.
```
Comment (0x84f) [inline news/2024/2024-05-03-5-digit-world-cup-2024-concludes.md:53]:
```suggestion
In the end, the run of a lifetime dissolved as Poland locked in and surged for victory, ultimately defeating the Australians in a 5-4 tiebreaker victory. The fact this match went to tiebreaker at all is incredible, and we're certain the Australian team is proud of their efforts.
```

File: news/2024/2024-05-03-5-digit-world-cup-2024-concludes.md [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+
+### Losers Grand Finals: ::{ flag=CA }:: Canada vs. ::{ flag=US }:: United States
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/p521LJbPT48" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Fighting for a chance at the crown or being cast to 3rd place, both teams initially won each of their picks in lockstep, but Canada's strides seemed more confident with their wider score margins. The even scoreline lasted even as the US began to show force in their Hard Rock picks.
+
+The first half of the match already revealed the stars of the show, with ::{ flag=CA }:: [ploot](https://osu.ppy.sh/users/7802400) carrying the Canadian team throughout their picks, and expecially so during HD2, [t+pazolite - Heartache Debug feat. Akagawa Nene](https://osu.ppy.sh/beatmapsets/1833716#osu/3764512), snatching the lobby's only S-rank score at 99.45% accuracy. This was not without a fight from ::{ flag=US }:: [gilgamesh815](https://osu.ppy.sh/users/14040810) and ::{ flag=US }:: [zfr](https://osu.ppy.sh/users/11536421), who also performed exceptionally well throughout.
```
Comment (0x84f) [inline news/2024/2024-05-03-5-digit-world-cup-2024-concludes.md:63]:
```suggestion
The first half of the match already revealed the stars of the show, with ::{ flag=CA }:: [ploot](https://osu.ppy.sh/users/7802400) carrying the Canadian team throughout their picks, and especially so during HD2, [t+pazolite - Heartache Debug feat. Akagawa Nene](https://osu.ppy.sh/beatmapsets/1833716#osu/3764512), snatching the lobby's only S-rank score at 99.45% accuracy. This was not without a fight from ::{ flag=US }:: [gilgamesh815](https://osu.ppy.sh/users/14040810) and ::{ flag=US }:: [zfr](https://osu.ppy.sh/users/11536421), who also performed exceptionally well throughout.
```

File: news/2024/2024-05-03-5-digit-world-cup-2024-concludes.md [hunk 6]
```diff
@@ -0,0 +1,255 @@
...
+
+Canada's slightly higher seed (#2 vs. #4) was first questioned when they almost lost on their own pick of [Hollow Wings](https://osu.ppy.sh/users/416662)'s famous map of [Halozy - Sentimental Skyscraper](https://osu.ppy.sh/beatmapsets/100348#osu/412288), falling behind yet taking it back on the second half. Carrying this momentum forward, Canada took the first breakpoint on America's HR3 pick, but traded it right back with America's double EZ roster on FM2.
+
+Another win from the US' own pick later and the two podium finalists were taken to the tiebreaker of [Sydosys - Epsilon](https://osu.ppy.sh/beatmapsets/2166904#osu/4571684). After a mostly smooth first section of the song, everyone breaks combo on the transition except for two Canadians, and with ::{ flag=CA }:: [FetherFall](https://osu.ppy.sh/users/15296550)'s subsequent break it was ::{ flag=CA }:: [Birchman](https://osu.ppy.sh/users/10676573)'s time to shine while building a devastating lead. The scoreline swung in Canada's favour, and with Birchman's 430k-score performance topping the lobby, Canada's advancement to Grand Finals against Poland was sealed in this moment.
```
Comment (0x84f) [inline news/2024/2024-05-03-5-digit-world-cup-2024-concludes.md:67]:
```suggestion
Another win from the US' own pick later and the two podium finalists were taken to the tiebreaker of [Sydosys - Epsilon](https://osu.ppy.sh/beatmapsets/2166904#osu/4571684). After a mostly smooth first section of the song, everyone breaks combo on the transition except for two Canadians, and with ::{ flag=CA }:: [FetherFall](https://osu.ppy.sh/users/15296550)'s subsequent break it was ::{ flag=CA }:: [Birchman](https://osu.ppy.sh/users/10676573)'s time to shine while building a devastating lead. The scoreline swung in Canada's favour, and with Birchman's 430k-score performance topping the lobby, Canada's advancement to Grand Finals against Poland was sealed at this moment.
```

File: news/2024/2024-05-03-5-digit-world-cup-2024-concludes.md [hunk 7]
```diff
@@ -0,0 +1,267 @@
...
+
+### Round of 16: ::{ flag=PL }:: Poland vs. ::{ flag=AU }:: Australia
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/qY9OcD7yj3A" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Moving into the Round of 16, we see what looks like an impossible matchup for Australia (#17). To move into Quarterfinals in the winners bracket, they must win against Poland (#1). The absurdity of this match cannot be understated.
+
+In the beginning, all looked normal for Poland, the 98.6% favourite according to the week's [pick'em challenge](/wiki/shared/news/2024-05-03-5-digit-world-cup-2024-concludes/pickem.png), as they took an easy two points from Australia. This is the behavior you expect from a #1 seed in the Round of 16, especially against a team who made it here via an upset against Sweden (#16).
```
Comment (0x84f) [inline news/2024/2024-05-03-5-digit-world-cup-2024-concludes.md:47]:
```suggestion
In the beginning, all looked normal for Poland, the 98.6% favourite according to the week's [pick'em challenge](/wiki/shared/news/2024-05-03-5-digit-world-cup-2024-concludes/pickem.png), as they took an easy two points from Australia. This is the behaviour you expect from a #1 seed in the Round of 16, especially against a team who made it here via an upset against Sweden (#16).
```

File: news/2024/2024-05-03-5-digit-world-cup-2024-concludes.md [hunk 8]
```diff
@@ -0,0 +1,267 @@
...
+
+### Losers Grand Finals: ::{ flag=CA }:: Canada vs. ::{ flag=US }:: United States
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/p521LJbPT48" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Fighting for a chance at the crown or being cast to 3rd place, both teams initially won each of their picks in lockstep, but Canada's strides seemed more confident with their wider score margins. The even scoreline lasted even as the US began to show force in their Hard Rock picks.
+
+The first half of the match already revealed the stars of the show, with ::{ flag=CA }:: [ploot](https://osu.ppy.sh/users/7802400) carrying the Canadian team throughout their picks, and expecially so during HD2, [t+pazolite - Heartache Debug feat. Akagawa Nene](https://osu.ppy.sh/beatmapsets/1833716#osu/3764512), snatching the lobby's only S-rank score at 99.45% accuracy. This was not without a fight from ::{ flag=US }:: [gilgamesh815](https://osu.ppy.sh/users/14040810) and ::{ flag=US }:: [zfr](https://osu.ppy.sh/users/11536421), who also performed exceptionally well throughout.
```
Comment (0x84f) [inline news/2024/2024-05-03-5-digit-world-cup-2024-concludes.md:63]:
```suggestion
The first half of the match already revealed the stars of the show, with ::{ flag=CA }:: [ploot](https://osu.ppy.sh/users/7802400) carrying the Canadian team throughout their picks, and especially so during HD2, [t+pazolite - Heartache Debug feat. Akagawa Nene](https://osu.ppy.sh/beatmapsets/1833716#osu/3764512), snatching the lobby's only S-rank score at 99.45% accuracy. This was not without a fight from ::{ flag=US }:: [gilgamesh815](https://osu.ppy.sh/users/14040810) and ::{ flag=US }:: [zfr](https://osu.ppy.sh/users/11536421), who also performed exceptionally well throughout.
```

File: news/2024/2024-05-03-5-digit-world-cup-2024-concludes.md [hunk 9]
```diff
@@ -0,0 +1,267 @@
...
+
+Canada's slightly higher seed (#2 vs. #4) was first questioned when they almost lost on their own pick of [Hollow Wings](https://osu.ppy.sh/users/416662)'s famous map of [Halozy - Sentimental Skyscraper](https://osu.ppy.sh/beatmapsets/100348#osu/412288), falling behind yet taking it back on the second half. Carrying this momentum forward, Canada took the first breakpoint on America's HR3 pick, but traded it right back with America's double EZ roster on FM2.
+
+Another win from the US' own pick later and the two podium finalists were taken to the tiebreaker of [Sydosys - Epsilon](https://osu.ppy.sh/beatmapsets/2166904#osu/4571684). After a mostly smooth first section of the song, everyone breaks combo on the transition except for two Canadians, and with ::{ flag=CA }:: [FetherFall](https://osu.ppy.sh/users/15296550)'s subsequent break it was ::{ flag=CA }:: [Birchman](https://osu.ppy.sh/users/10676573)'s time to shine while building a devastating lead. The scoreline swung in Canada's favour, and with Birchman's 430k-score performance topping the lobby, Canada's advancement to Grand Finals against Poland was sealed in this moment.
```
Comment (0x84f) [inline news/2024/2024-05-03-5-digit-world-cup-2024-concludes.md:67]:
```suggestion
Another win from the US' own pick later and the two podium finalists were taken to the tiebreaker of [Sydosys - Epsilon](https://osu.ppy.sh/beatmapsets/2166904#osu/4571684). After a mostly smooth first section of the song, everyone breaks combo on the transition except for two Canadians, and with ::{ flag=CA }:: [FetherFall](https://osu.ppy.sh/users/15296550)'s subsequent break it was ::{ flag=CA }:: [Birchman](https://osu.ppy.sh/users/10676573)'s time to shine while building a devastating lead. The scoreline swung in Canada's favour, and with Birchman's 430k-score performance topping the lobby, Canada's advancement to Grand Finals against Poland was sealed at this moment.
```

File: news/2024/2024-05-03-5-digit-world-cup-2024-concludes.md [hunk 10]
```diff
@@ -0,0 +1,267 @@
...
+
+Lagging behind and desperately in need of momentum, Poland picked DT3. After an uncertain ending with ::{ flag=AU }:: [Trainjumper](https://osu.ppy.sh/users/2272587)'s client freezing on the livestream, the results were revealed in the [match logs](https://osu.ppy.sh/community/matches/113190844). Australia won again and now had the advantage at 2-3. They shot for NM2 and won yet another point, somehow flipping the script and making Australia the one with match point against the would-be winners of the tournament.
+
+In the end, the run of a lifetime dissolved as Poland locked in and surges for victory, ultimately defeating the Australians in a 5-4 tiebreaker victory. The fact this match went to tiebreaker at all is incredible, and we're certain the Australian team is proud of their efforts.
```
Comment (0x84f) [inline news/2024/2024-05-03-5-digit-world-cup-2024-concludes.md:53]:
```suggestion
In the end, the run of a lifetime dissolved as Poland locked in and surged for victory, ultimately defeating the Australians in a 5-4 tiebreaker victory. The fact this match went to tiebreaker at all is incredible, and we're certain the Australian team is proud of their efforts.
```

File: news/2024/2024-05-03-5-digit-world-cup-2024-concludes.md [hunk 11]
```diff
@@ -0,0 +1,267 @@
...
+</div>
+
+Moving into the Round of 16, we see what looks like an impossible matchup for Australia (#17). To move into Quarterfinals in the winners bracket, they must win against Poland (#1). The absurdity of this match cannot be understated.
+
+In the beginning, all looked normal for Poland, the 98.6% favourite according to the week's [pick'em challenge](/wiki/shared/news/2024-05-03-5-digit-world-cup-2024-concludes/pickem.png), as they took an easy two points from Australia. This is the behavior you expect from a #1 seed in the Round of 16, especially against a team who made it here via an upset against Sweden (#16).
+
+As the match charges on, Australia managed to earn point after point. Expecting an easy 3-0 lead, Poland picked HR3 and lost by a huge margin (by nearly 500k points). The score was now 2-1 and the ball was in Australia's court. Looking to tie things up, the Aussies went for FM2 and managed to win by 300k - a second victory in a row over the goliath that is the Polish team. Everyone's pick'ems were already shattered by this point, but the Australian team still had more up their sleeves and the score is tied 2-2.
```
Comment (0x84f) [inline news/2024/2024-05-03-5-digit-world-cup-2024-concludes.md:49]:
```suggestion
As the match charges on, Australia managed to earn point after point. Expecting an easy 3-0 lead, Poland picked HR3 and lost by a huge margin (by nearly 500k points). The score was now 2-1 and the ball was in Australia's court. Looking to tie things up, the Aussies went for FM2 and managed to win by 300k — a second victory in a row over the goliath that is the Polish team. Everyone's pick'ems were already shattered by this point, but the Australian team still had more up their sleeves and the score is tied 2-2.
```

**Review by peppy — DISMISSED** [review]

**Review by Hiviexd — APPROVED** [review]
good read

**Pull request merged**